web/web.go

```diff
@@ -201,7 +202,7 @@ func New(o *Options) *Handler {
 
 	router.Get("/heap", readyf(instrf("heap", dumpHeap)))
 
-	router.Get(o.MetricsPath, readyf(prometheus.Handler().ServeHTTP))
+	router.Get(o.MetricsPath, readyf(promhttp.Handler().ServeHTTP))
```

Comment:
No, we can't do this here, the new HTTP handler has new semantics and is not a 1:1 replacement. This should be a 2.0 change.

Reply:
Sounds good.